bin/tools.py

```diff
@@ -505,28 +496,7 @@ def build_parser():
     genparser.add_argument(
         '--timeout', '-t', type=int, help='Override the default timeout. Default: 30.'
     )
-    genparser.add_argument(
-        '--samples',
-        action='store_true',
-        help='Overwrite the samples as well, in combination with -f.',
-    )
-
-    genparser_group = genparser.add_mutually_exclusive_group()
-    genparser_group.add_argument(
```

Comment:
The one benefit of `--add-unlisted` is that one can just dump files in `data/secret`, run `bt generate --add-unlisted` and then they're automatically moved to `generators/manual/` *and* a rule in `generators.yaml` is added.
Without this command, each new testcase needs two actions: 1) writing the file, and 2) adding the name of the file to `generators.yaml`, which is slightly redundant.

Nevertheless, I suppose we're all getting more and more used to `bt generate` now and the new workflow is not terrible, and also rarely used.
If needed we could add a new command `bt add_manual_generated_files generators/manual/*.in` which updates the yaml.

Reply:
We can add a separate command but the workflow should be:

- dump the files in `generator/???`
- add the rules to the `generator.yaml` with the new commnd

but moving from `data` is just weird since you are not supposed to touch `data`?

Reply:
Yeah I agree; I'm just used to the old behaviour :P Removing and cleaning up is good :)